controllers/ovncontroller_controller.go

```diff
@@ -248,7 +248,8 @@ func (r *OVNControllerReconciler) findObjectsForSrc(ctx context.Context, src cli
 		}
 		err := r.Client.List(ctx, crList, listOps)
 		if err != nil {
-			return []reconcile.Request{}
+			Log.Error(err, fmt.Sprintf("listing %s for field: %s - %s", crList.GroupVersionKind().Kind, field, src.GetNamespace()))
+			return requests
```

Comment:
due to the loop over allWatchFields there can already be requests from a previous run to return

Reply:
Right. But I was confused about whether it's intentional since the PR title talks about log message.

---

Question: before this PR, when we did not return any pending requests, were we losing some updates in this error scenario?

Reply:
yes, thats what we hit in telemetry. when loop over the fields we got reconcile requests from the first ones, on later ones there was an error and we just returned an empty list and no reconcile happened.